### PR TITLE
Fix for notification for muted conversation

### PR DESCRIPTION
**Pull request by joonhocho**
@2is10 @andrewconner 

I am not sure why `(n.unreadAuthors > 0)` is necessary, but the line is the source of the bug. Tested with and without the code and confirmed.

### CODE REVIEW

**Comment by joonhocho** [comment]
Please merge this request if this seems to be the right way to fix the bug and there will be no side effect.
@2is10  @andrewconner

File: packrat/main.js [hunk 1]
```diff
@@ -929,7 +929,6 @@ function removeNotificationPopups(associatedId) {
 
 function standardizeNotification(n) {
   n.category = (n.category || "message").toLowerCase();
-  n.unread = n.unread || (n.unreadAuthors > 0);
```
Comment (2is10) [inline packrat/main.js:932]: This line was added for backwards compatibility. The `.unread` field is being made obsolete (redundant) by `.unreadAuthors`, which identifies how many authors have added messages that the user has not yet read. The `.unreadAuthors` field will be used to bold the names of conversation participants that have added messages to the thread that the viewer has not yet read.

If the semantics of muting is that messages are automatically marked as read, then @stkem will need to change Eliza to omit `.unreadAuthors` (or set it to `0`) for muted threads. We should not remove this line of code from the extension.

My opinion, by the way, is that we should not conflate muting and read/unread state. I’ll discuss this separately with Effi. In the meantime, Eliza should behave as spec’d (treating new messages on muted threads as read).

**Comment by 2is10** [comment]
Thanks for looking into this issue, Joon, and sorry you were pulled into an issue that Stephen and I caused.

**Comment by joonhocho** [comment]
No prob. I agree that we should have separate muted flag since it is very confusing to use unread as replacement.

**Pull request closed without merging**